src/utils/ssh.ts

```
@@ -1,121 +1,119 @@
+          } else {
+            reject(new Error('SSH session failed with code ' + code));
+          }
+        });
+        ssh.on('error', reject);
       });
-      ssh.on('error', reject);
-    });
+    }
   }
-}
+);
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)